From f3820f1a9837451b9209aa40ca5ab750e59fec19 Mon Sep 17 00:00:00 2001
From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Thu, 2 Oct 2014 10:18:11 +0200
Subject: [PATCH] mohqueue: fixed db_qtable parameter name

- reported by Marino Mileti

(cherry picked from commit 321575dcd18e42fe6c62942ef417dd8acf7f466d)

Conflicts:
	modules/mohqueue/mohq.c
---
 modules/mohqueue/mohq.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/mohqueue/mohq.c b/modules/mohqueue/mohq.c
index 0c0d118..20a9ace 100644
--- a/modules/mohqueue/mohq.c
+++ b/modules/mohqueue/mohq.c
@@ -70,7 +70,7 @@ int moh_maxcalls = 50;
 static param_export_t mod_parms [] = {
   { "db_url", STR_PARAM, &db_url },
   { "db_ctable", STR_PARAM, &db_ctable },
-  { "db_ctable", STR_PARAM, &db_qtable },
+  { "db_qtable", STR_PARAM, &db_qtable },
   { "mohdir", STR_PARAM, &mohdir },
   { "moh_maxcalls", INT_PARAM, &moh_maxcalls },
   { NULL, 0, NULL },
@@ -439,4 +439,4 @@ if (pmod_data->pcall_lock->plock)
 shm_free (pmod_data);
 pmod_data = NULL;
 return -1;
-}
\ No newline at end of file
+}
-- 
2.1.1

